src/app/styles.ts

```diff
@@ -220,6 +220,7 @@ export const globalViewDark = css`
 
   .react-flow {
     flex: 1;
+    height: 75vh;
```

Comment:
You introduced it again ...

This one should be removed / replaced by a better mechanism.